shared-libs/transitions/src/schedule/due_tasks.js

```diff
@@ -51,9 +50,15 @@ module.exports = {
           return callback(err);
         }
 
-        var objs = _.unique(result.rows, false, function(row) {
-          return row.id;
-        });
+        const objs = result.rows.reduce((objs, row) => {
+          if (!objs[row.id]) {
+            row.keys = [];
+            objs[row.id] = row;
+          }
+          objs[row.id].keys.push(row.key);
+
+          return objs;
+        }, {});
```

Comment:
Fix for updating all valid `scheduled_tasks` instead of one.

Reply:
I know this is a pain but can we separate this out as a separate fix? It'd make it much easier to find later. It can wait until this PR is merged.

Reply:
Sure.

Reply:
Changes have been reverted.